spyder/widgets/variableexplorer/dataframeeditor.py

```diff
@@ -399,23 +401,102 @@ def reset(self):
         self.endResetModel()
 
 
+class FrozenTableView(QTableView):
+    def __init__(self, parent):
+        QTableView.__init__(self, parent)
+        self.setModel(parent.model())
+        self.setFocusPolicy(Qt.NoFocus)
+        self.verticalHeader().hide()
+        if PYQT5:
+            self.horizontalHeader(). \
+            setSectionResizeMode(QHeaderView.Fixed)
+        else:
+            self.horizontalHeader(). \

```

Comment:
Let's not add continuation lines to these statements (even if they are longer than 79 columns)